templates/terraform/command-center-project/service-accounts.tf

```diff
@@ -10,3 +10,24 @@ module command_center_gke_runner_account {
   vault_path = "${local.vault_prefix}/service-accounts/gke-runner"
   roles = ["logging.logWriter", "monitoring.metricWriter", "monitoring.viewer"]
 }
+
+module v2f_writer {
+  source = "/templates/google-sa"
+  providers = {
+    google.target = google.target,
+    vault.target = vault.target
+  }
+
+  account_id = "v2f-writer"
+  display_name = " Service account to write V2F data to GCS"
+  vault_path = "${local.vault_prefix}/service-accounts/v2f-writer"
+  roles = []
+}
```

Comment:
I wouldn't put this here. Instead, I'd include it in the `v2f-prod` environment definition.

Reply:
WELP I didn't even realize we had that, eyes totally glossed over it, ty